### DISCUSSION TITLE
Translations into Spanish and Galician

### DISCUSSION THREAD

**Discussion by cabaseira** [urania-dev/snapp#24] [Show and tell]
Thank you for your fantastic application.
I have created some files to translate it into Spanish (Spain) and Galician (Spain) that I share with you.

Español (España) | es-ES 
[es.json](https://github.com/urania-dev/snapp/files/15293312/es.json)

Galego (España) | gl-ES
[gl.json](https://github.com/urania-dev/snapp/files/15293313/gl.json)

**Comment by urania-dev** [comment]
Thank you, i will integrate them in next release :)

**Comment by urania-dev** [comment]
i tried to port your translations in the new version, but ended up rounding all the i18n with gpt ^^" 

i'm aware of issues here and there, i didn't meant to disregard your contribute but i couldn't manage anyway rebuilding from the top.

for what is worth the file is more readable now

**Comment by cabaseira** [comment]
Glad to have contributed anyway. Thank you for your work.

**Reply by urania-dev** [reply]
thank you! still kept the galician tho :)
